config.mcp.toml

```diff
@@ -0,0 +1,17 @@
+# MCP Server Configuration
```

Comment:
@openhands remove config.mcp.toml and docs/mcp-server.md. Investigate how the MCP config is plumbed into the agent currently. Modify it at runtime to include the server running a http://localhost:3000/mcp, and make the host name configurable via an env var

Reply:
@openhands try again